src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -5854,6 +5864,27 @@ namespace CondenserLoopTowers {
             }
             if (this->WaterTemp > (TempSetPoint + state.dataCondenserLoopTowers->towers(this->VSTower).MaxRangeTemp)) { // run flat out
                 OutletWaterTempLocal = this->WaterTemp - state.dataCondenserLoopTowers->towers(this->VSTower).MaxRangeTemp;
+            } else {
+                if (this->WaterTemp > TempSetPoint) {
+                    ++this->VSErrorCountTRCalc;
+                    if (this->VSErrorCountTRCalc < 2) {
+                        ShowWarningError(
+                            state,
+                            format("The outlet water temperature for cooling tower {} could not be correctly calculated. The outlet water "
+                                   "temperature was set to the inlet water temperature minus 0.001 deg. C.",
+                                   this->Name));
+                        ShowContinueError(state,
+                                          format(" ... Occurrence info = {}, {} {}",
+                                                 state.dataEnvrn->EnvironmentName,
+                                                 state.dataEnvrn->CurMnDy,
+                                                 General::CreateSysTimeIntervalString(state)));
+                    } else {
+                        ShowRecurringWarningErrorAtEnd(
+                            state,
+                            format("The outlet water temperature for cooling tower {} could not be calculated.", this->Name),
+                            this->ErrIndexTRCalc);
+                    }
+                }
```

Comment:
New warning to identify cases where we essentially don't do anything. This situation _should_ not arise anymore but this would catch it in case it occurs.